base/actions/actions.py

```diff
@@ -493,6 +546,25 @@ def validate_tracing(
     ) -> Dict[Text, Optional[Text]]:
         return self.validate_generic("tracing", dispatcher, value, self.yes_no_data)
 
+    def get_healthcheck_data(self, tracker: Tracker, risk: Text) -> Dict[Text, Any]:
+        return {
+            "deduplication_id": uuid.uuid4().hex,
+            "msisdn": f'{tracker.sender_id.lstrip("+")}',
+            "source": "WhatsApp",
+            "province": f'ZA-{tracker.get_slot("province").upper()}',
+            "city": tracker.get_slot("location"),
+            "age": self.AGE_MAPPING[tracker.get_slot("age")],
+            "gender": self.GENDER_MAPPING[tracker.get_slot("gender")],
+            "cough": tracker.get_slot("symptoms_cough").replace(" ", "_"),
+            "fever": self.YES_NO_MAPPING[tracker.get_slot("symptoms_fever")],
+            "sweat": self.YES_NO_MAPPING[tracker.get_slot("symptoms_sweat")],
+            "weight": self.YES_NO_MAPPING[tracker.get_slot("symptoms_weight")],
+            "exposure": self.YES_NO_MAYBE_MAPPING[tracker.get_slot("exposure")],
+            "tracing": self.YES_NO_MAPPING[tracker.get_slot("tracing")],
+            "risk": risk,
+            "location": self.fix_location_format(tracker.get_slot("location_coords")),
```

Comment:
Do we need the `fix_location_format`? Can we not just put `location_coords` directly here?